core/ir/aarch64/instr_create_api.h

```diff
@@ -658,14 +658,49 @@
     instr_create_0dst_3src((dc), OP_tbnz, (pc), (reg), (imm))
 #define INSTR_CREATE_cmp(dc, rn, rm_or_imm) \
     INSTR_CREATE_subs(dc, OPND_CREATE_ZR(rn), rn, rm_or_imm)
-#define INSTR_CREATE_eor(dc, d, s)                                      \
-    INSTR_CREATE_eor_shift(dc, d, d, s, OPND_CREATE_INT8(DR_SHIFT_LSL), \
-                           OPND_CREATE_INT8(0))
+
+/**
+ * Creates an EOR instruction with one output and two inputs. For simplicity, the first
+ * input reuses the output register.
+ *
+ * \param dc        The void * dcontext used to allocate memory for the instr_t.
+ * \param d         The output register and the first input register.
+ * \param s_or_imm  The second input register or immediate.
+ */
+#define INSTR_CREATE_eor(dc, d, s_or_imm)                                            \
```

Comment:
Please list these 2 added macros and 2 updated macros in the PR description == final commit message.

Reply:
Done.